src/plugins/ResizeReorder/ResizerLine.js

```diff
@@ -112,38 +117,52 @@ class ColumnResizerLine extends React.PureComponent {
     );
   }
 
+  componentWillReceiveProps(nextProps) {
+    if(nextProps.instance){
+      this.state.top = nextProps.instance.getBoundingClientRect().top;
+      this.state.left = nextProps.instance.getBoundingClientRect().left;
+      this.state.knobWidth = nextProps.instance.getBoundingClientRect().width;
```

Comment:
This is needed to position the `ResizerLine`, since it has been portalled to the end of the DOM.

Reply:
Don't mutate state. Use `this.setState` instead

Reply:
I need a mutable variable, hence I shall migrate `top`, `left` and `knobWidth` to another variable `instanceDetails`.